crates/standalone/src/lib.rs

```diff
@@ -653,7 +653,7 @@ impl StandaloneEnv {
             .context("failed to load module program")?
             .ok_or_else(|| anyhow!("missing object: {}", database.program_bytes_address.to_hex()))?;
 
-        let root_db_path = stdb_path("worker_node/database_instances");
+        let root_db_path = stdb_path("data/worker_node/database_instances");
```

Comment:
This is a breaking change, yes? Could you please add to the PR description an exhaustive list of all the files that you expect to be in the `.spacetime` directory. 

Might need to get Kim's buy in because this could also break cloud maybe?

Reply:
Will make a proposal for this 👍